system/services/assetManager/NativeImageService.cfc

```diff
@@ -193,7 +194,24 @@ component displayname="Native Image Manipulation Service" {
 			}
 		}
 
-		pdf attributeCollection=pdfAttributes;
+		if ( SERVER.lucee.version.left(1) gt 4 ) {
```

Comment:
Extra safety checking:

```
if ( Val( Left( server.lucee.version ?: "", 1 ) ) > 4 ) {...
```

`Val()` ensures any text we have is a number valid for numeric comparison. Adding elvis here also to protect against any other unexpected changes to that variable that we can't guarantee is set (even though it isn't likely to change).